lectures/opt_savings.md

```diff
@@ -0,0 +1,455 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Optimal Savings
+
+In addition to what’s in Anaconda, this lecture will need the following libraries:
+
+```{code-cell} ipython3
+!pip install quantecon
+```
+
+We will use the following imports:
+
+```{code-cell} ipython3
+import numpy as np
+import quantecon as qe
```

Comment:
Can you please try to remove `np.*` from this lecture - and use `jnp.*` instead?